README.md

```diff
@@ -67,25 +67,60 @@ For a full list of available tools please see the [tools section](http://fulcrum
 
 Below we highlight a few tools that you may find useful.
 
-* Tools for working with Unique Molecular Indexes (UMIs, aka Molecular IDs or MIDs). 
-  * Annotating/Extract Umis from read-level data: `AnnotateBamWithUmis` and `ExtractUmisFromBam`.
-  * Tools to manipulate read-level data containing Umis: `CorrectUmis`, `GroupReadsByUmi`, `CallMolecularConsensusReads` and `CallDuplexConsensusReads`
-* Tools to manipulate read-level data:
-  * FastqManipulation: `DemuxFastqs` and `FastqToBam`
-  * Filter read-level data: `FilterBam`.
-  * Clipping of reads: `ClipBam`.
-  * Randomize the order of read-level data: `RandomizeBam`.
-  * Update read-level metadata: `SetMateInformation` and `UpdateReadGroups`.
-* Quality assessment tools:
-  * Detailed substitution error rate evaluation: `ErrorRateByReadPosition`
-  * Sample pooling QC: `EstimatePoolingFractions`
-  * Splice-aware insert size QC for RNA-seq libraries: `EstimateRnaSeqInsertSize`
-  * Assessment of duplex sequencing experiments: `CollectDuplexSeqMetrics`
-* Miscellaneous tools:
-  * Pick molecular indices (ex. sample barcodes, or molecular indexes): `PickIlluminaIndices` and `PickLongIndices`.
-  * Convert the output of HAPCUT (a tool for phasing variants): `HapCutToVcf`.
-  * Find technical or synthetic sequences in read-level data: `FindTechnicalReads`.
-  * Assess phased variant calls: `AssessPhasing`.
+-   Tools for working with Unique Molecular Indexes (UMIs, aka Molecular IDs or Molecular Barcodes):
+    -   Annotating/Extract Umis from read-level data: [`FastqToBam`][fgbio-fastqtobam-link], [`AnnotateBamWithUmis`][fgbio-annotatebamwithumis-link], [`ExtractUmisFromBam`][fgbio-extractumisfrombam-link], and [`CopyUmiFromReadName`][fgbio-copyumifromreadname-link].
```

Comment:
```suggestion
    -   Annotate/Extract Umis from read-level data: [`FastqToBam`][fgbio-fastqtobam-link], [`AnnotateBamWithUmis`][fgbio-annotatebamwithumis-link], [`ExtractUmisFromBam`][fgbio-extractumisfrombam-link], and [`CopyUmiFromReadName`][fgbio-copyumifromreadname-link].
```

I think the tense should be consistent here. `Annotating/Extracting` would also be fine by me, but the rest of the docs mostly use imperative tense. Up to you 🙂